source/distributed/tria.cc

```diff
@@ -3201,7 +3198,11 @@ namespace parallel
     DEAL_II_CXX20_REQUIRES((concepts::is_valid_dim_spacedim<dim, spacedim>))
     std::vector<types::subdomain_id> Triangulation<dim, spacedim>::
       find_point_owner_rank(
-        [[maybe_unused]] const std::vector<Point<dim>> &points)
+#  ifndef P4EST_SEARCH_LOCAL
+        const std::vector<Point<dim>> &/*points*/
+#  else
+        const std::vector<Point<dim>> &points)
+#  endif
```

Comment:
My preference is to either just bring back the `void` cast for this case, or to do `std::ignore = points`. Using the postprocessor is just too ugly and distracting, in particular because it expresses no functional difference here.